docs/guide/choose-azure-container-service.md

```diff
@@ -74,17 +74,24 @@ For a general introduction into the terminology and concepts around service mode
 
 ### Comparing the service models of Azure container solutions
 
-#### AKS
+#### AKS Standard
 
-As a hybrid of IaaS and PaaS, AKS prioritizes control over simplicity. Though AKS streamlines the management of the underlying core infrastructure, this VM-based platform is still exposed to your applications and requires appropriate guardrails and processes, like patching, to ensure security and business continuity. The compute infrastructure is supported by additional Azure resources that are hosted directly in your subscription, like Azure load balancers.
+As a hybrid of IaaS and PaaS, AKS prioritizes control over simplicity, leveraging the de facto standard for container orchestration: Kubernetes. Though AKS streamlines the management of the underlying core infrastructure, this VM-based platform is still exposed to your applications and requires appropriate guardrails and processes, like patching, to ensure security and business continuity. The compute infrastructure is supported by additional Azure resources that are hosted directly in your subscription, like Azure load balancers.
 
 AKS also provides access to the Kubernetes API server, which enables you to customize the container orchestration and thus deploy projects from the Cloud Native Computing Foundation (CNCF). Consequently, there's a significant learning curve for workload teams that are new to Kubernetes. If you're new to containerized solutions, this learning curve must be taken in consideration. The following PaaS solutions offer a lower barrier to entry. You can move to Kubernetes when your requirements dictate that move.
 
 #### AKS Automatic
 
+[AKS Automatic](https://learn.microsoft.com/en-us/azure/aks/intro-aks-automatic) simplifies the adoption of Kubernetes by automating complex cluster management tasks, reducing the need for deep Kubernetes expertise. It offers a more streamlined, PaaS-like experience while retaining the flexibility and extensibility of Kubernetes. Azure handles cluster setup, node provisioning, scaling, security patching, and applies best-practice configurations by default. This reduces operational overhead and accelerates time-to-market. 
```

Comment:
Can we change "This reduces operational **overhead**" to "This reduces operational **effort**"?

Reply:
Another thing: does AKS Automatic simplifies only at the deployment phase by pre-selecting default settings or does it also lockdown those configurations after deployment?

Reply:
Yes, it needs to be clear that after AKS Automatic does its job, it is up to the operator to make subsequent changes through the API. So the operator needs to be comfortable with that if they choose AKS Automatic.

Reply:
**From Andre (We'll be moving forward with this)**:

To know what allowed and what's not, check this doc page tables: https://learn.microsoft.com/en-us/azure/aks/intro-aks-automatic
 
Pre-configured = features are always enabled and you can't disable or change their settings
Default = features are configured for you but can be changed
Optional = features are available for you to configure and are not enabled by default
 
This list will probably change very often so I think it would be better to not mention the specific settings in our guide but rather point to that page